src/components/artistas/ArtistaPortfolio.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { toastr } from 'react-redux-toastr';
+
+import Uploads from '../fields/Uploads';
+import { ToastrOptionsConfirm } from '../../utilities/toastr.js';
+
+class ArtistaPortfolio extends Component {
+  state = {
```

Comment:
this is just being used to manage the values in the add item form right?

Reply:
Yes, the component state